app/src/migrations/20200424000000-1-col-name-col-nulls.js

```diff
@@ -0,0 +1,97 @@
+        queryInterface.changeColumn('contact', 'phone1', {
+          type: Sequelize.STRING,
+          allowNull: true,
+          transaction: t
+        }),
+        queryInterface.changeColumn('contact', 'email', {
+          type: Sequelize.STRING,
+          allowNull: true,
+          transaction: t
+        }),
+        queryInterface.changeColumn('location', 'startDate', {
+          type: Sequelize.DATE,
+          allowNull: true,
+          transaction: t
+        }),
+        queryInterface.changeColumn('location', 'endDate', {
+          type: Sequelize.DATE,
+          allowNull: true,
+          transaction: t
+        }),
+        queryInterface.changeColumn('location', 'addressLine1', {
+          type: Sequelize.STRING,
+          allowNull: true,
+          transaction: t
+        }),
+        queryInterface.changeColumn('location', 'city', {
+          type: Sequelize.STRING,
+          allowNull: true,
+          transaction: t
+        }),
+        queryInterface.changeColumn('location', 'province', {
+          type: Sequelize.STRING,
+          allowNull: true,
+          transaction: t
+        }),
+        queryInterface.changeColumn('location', 'postalCode', {
+          type: Sequelize.STRING,
+          allowNull: true,
+          transaction: t
+        })
+      ]);
+    });
+  },
+
+  down: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.renameColumn('ipc_plan', 'distancingMaintained', 'distancingSleepingBarriers', {transaction: t}),
+        queryInterface.changeColumn('contact', 'phone1', {
+          type: Sequelize.STRING,
+          allowNull: false,
+          transaction: t
+        }),
+        queryInterface.changeColumn('contact', 'email', {
+          type: Sequelize.STRING,
+          allowNull: false,
+          transaction: t
+        }),
+        queryInterface.changeColumn('location', 'startDate', {
+          type: Sequelize.DATE,
+          allowNull: false,
+          transaction: t
+        }),
+        queryInterface.changeColumn('location', 'endDate', {
+          type: Sequelize.DATE,
+          allowNull: false,
+          transaction: t
+        }),
+        queryInterface.changeColumn('location', 'addressLine1', {
+          type: Sequelize.STRING,
+          allowNull: false,
+          transaction: t
+        }),
+        queryInterface.changeColumn('location', 'city', {
+          type: Sequelize.STRING,
+          allowNull: false,
+          transaction: t
+        }),
+        queryInterface.changeColumn('location', 'province', {
+          type: Sequelize.STRING,
+          allowNull: false,
+          transaction: t
+        }),
+        queryInterface.changeColumn('location', 'postalCode', {
+          type: Sequelize.STRING,
+          allowNull: false,
+          transaction: t
+        })
+      ]);
+    });
```

Comment:
I know this may be pedantic, but we should make sure that this array ordering is the exact inverse of the up function's equivalent. I know that this will work fine given the context, but we should try to keep reverse ordering in place to prevent any chance of it breaking.